### PR TITLE
Condense PR template

### PR DESCRIPTION
**Pull request by warp-core**
**Documentation**

## Summary
The PR template currently has four sections: content, bugfix, balance, and feature.
Each of these sections has a series of subsections. With the idea being that a user should choose one section and then fill in each subsection, which should be relevant to that section.
In practice, there is a lot of overlap in terms of subsections between each section.
For example, all but the bugfix section begin with a "Summary" subsection. The bugfix section beginning with a "Fix Details" subsection, but this is essentially the same as the summary subsections of every other section. The only difference being the name.
All but the feature section have a "Save File" subsection, however, it is perfectly possible for a feature PR to benefit from the inclusion of an appropriate save file to test it.
Similarly, the bugfix and feature sections contain "Testing Done" subsections, but there's no reason that content and balance PRs should be excluded from describing testing done.

This PR condenses the template into a single section. A list of headings is provided at the top (a user may choose to provide their own instead of choosing one of the examples if they prefer.) Below, are many subsections for the user to fill out as appropriate.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,67 +1,38 @@
-NOTICE: Delete the sections that do not apply to your PR, and fill out the section that does.
-(You can open a PR to add or improve a section, if you find them lacking!)
+(The lines in brackets (like this one) is instructions for filling out this template. These lines can be deleted.)
```
Comment (bene-dictator) [inline .github/pull_request_template.md:1]:
```suggestion
(The lines in brackets (like this one) are instructions for filling out this template. These lines can be deleted.)
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,67 +1,38 @@
...
+{{Include before and after screenshots demonstrating your changes, if applicable.}}
 
-## Fix Details
-{{add details}}
+## Usage examples
+{{If your changes affect how game data can be defined, provide examples demonstrating the changes here.}}
 
 ## Testing Done
-{{describe how you tested that the fix doesn't introduce other issues}}
+{{Describe how you tested these changes. How they meet the purpose. Ensure they don't introduce new issues.}}
```
Comment (bene-dictator) [inline .github/pull_request_template.md:25]: What does "how they meet the purpose" mean? Is it in relation to the issue this PR addresses or whether the test itself works?

**Review by bene-dictator — COMMENTED** [review]
~~Now I have to pay attention to the template instead of deleting everything below bugfix. shaking my smh.~~

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,67 +1,38 @@
...
-
-## Usage Examples
-{{if this feature is used in the data files, provide examples!}}
-
-## Testing Done
-{{describe how you tested the new feature}}
-
-### Automated Tests Added
-{{describe the automated tests you added (using the unit-test framework, integration-test framework or otherwise) to reduce the risk of regressions in the future. "N/A" if this PR is not for an in-game feature that needs to remain functional in the future. }}
+## Artwork Checklist
```
Comment (Amazinite) [inline .github/pull_request_template.md:32]:
```suggestion
## Artwork Checklist
{{If any artwork was added or changed by this PR, the following must be provided.}}
```

File: .github/pull_request_template.md (hunk 2 above)
Comment (Amazinite) [reply .github/pull_request_template.md:25]:
```suggestion
{{Describe how you tested these changes. Ensure that new issues aren't introduced.}}
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (Amazinite) [reply .github/pull_request_template.md:1]:
```suggestion
(The lines in parentheses (like this one) are instructions for filling out this template. These lines can be deleted.)
```

**Review by Amazinite — APPROVED** [review]

**Pull request merged**